### DISCUSSION TITLE
Is there a function calculating minimum rotated rectangle?

### DISCUSSION THREAD

**Discussion by DMuzer** [NetTopologySuite/NetTopologySuite#625] [Q&A] [answered]
I used python shapely package, as I found shapely is based on same engine as NTS. There was a function calculated minimum rotated rectlangle, but I cant find similar funtion in NTS. Is there this function in NTS?

**Comment by DGuidi** [comment, marked as answer]
I think [MinimumDiameter.GetMinimumRectangle](https://github.com/NetTopologySuite/NetTopologySuite/blob/develop/src/NetTopologySuite/Algorithm/MinimumDiameter.cs#L292) does the same think

**Reply by DMuzer** [reply]
Thanks! This is it!
